src/plays/steps/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
```

Comment:
Please avoid targeting directly.

Reply:
Since App.css in the src folder has the same style applied for all elements I have removed this style. Also removing these styles made no difference to the project so I believe it is safe to remove.